POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/App.xaml.cs

```diff
@@ -28,17 +28,49 @@ public App()
 
         protected override async void OnLaunched(LaunchActivatedEventArgs args)
         {
+            /// <summary>
+            /// Sandbox settings is a way to emulate background notifications on install and update
+            /// Not to be inclued when added to final product
+            /// </summary>
+            SandboxNotifications();
+            
             if (!args.PrelaunchActivated)
             {
                 await ActivationService.ActivateAsync(args); 
             }
+           
+        }
 
-            NotificationService.AppInstalledToast();
-            NotificationService.AppUpdatedToast();
+        /// <summary>
+        /// These notifications should be activated in conjunction with new app installs or updates
+        /// </summary>
+        private void SandboxNotifications()
+        {
+            var lSettings = ApplicationData.Current.LocalSettings;
+            lSettings.Values["IsFirstRun"] = true;
+            lSettings.Values["NewVersion"] = "1.0.0.1"; // Reference to the newest version just installed
```

Comment:
should we be hard coding version numbers?

Reply:
Switched to SystemInformation flags to fix this issue